Makefile

```diff
@@ -636,42 +638,36 @@ run-c-tests: all-c-tests
 	@echo '---- Run C unit-tests: ----'
 	./$(C_UNIT_TEST_BIN)
 	@echo
-	./$(L3_DUMP)
+	python3 $(L3_DUMP) $(L3_DUMP_ARG_LOG_FILE) $(L3_C_DATA) $(L3_DUMP_ARG_BINARY) ./$(C_UNIT_TEST_BIN)
```

Comment:
Replace all instances of calls to this tool to now specify the required command-line flags.